### PR TITLE
ADM-987 [backend]: adjust the logs for share

### PR DESCRIPTION
**Pull request by zhou-yinyuan**
## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

File: backend/src/main/java/heartbeat/repository/FileRepository.java [hunk 1]
```diff
@@ -174,10 +184,12 @@ public List<String> getFiles(FileType fileType, String uuid) {
 		File folder = new File(fileName);
 
 		if (folder.exists() && folder.isDirectory()) {
+			log.info("Successfully get the {} folder in the report files", fileName);
 			return Arrays.stream(folder.listFiles()).map(File::getName).toList();
 		}
 		else {
-			throw new NotFoundException(String.format("Don't find the %s folder in the report files", uuid));
+			log.error("Don't find the {} folder in the report files", fileName);
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/repository/FileRepository.java:191]: `Failed to find the {} folder in the report files", fileName);`

File: backend/src/main/java/heartbeat/service/report/ReportService.java [hunk 2]
```diff
@@ -99,14 +101,23 @@ private boolean isNotGenerateMetricError(ReportMetricsError reportMetricsError)
...
+			reportUrls = fileRepository.getFiles(FileType.REPORT, uuid)
+				.stream()
+				.map(it -> it.split(FILENAME_SEPARATOR))
+				.filter(it -> it.length > 2)
+				.map(it -> this.generateReportCallbackUrl(uuid, it[1], it[2]))
+				.distinct()
+				.toList();
+		}
+		catch (NotFoundException e) {
+			log.error("Get share details result: failed, reportId: {}", uuid);
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/service/report/ReportService.java:115]: Failed to ...

File: backend/src/main/java/heartbeat/service/report/ReportService.java [hunk 3]
```diff
@@ -99,14 +101,23 @@ private boolean isNotGenerateMetricError(ReportMetricsError reportMetricsError)
...
+			throw new NotFoundException(String.format("Don't find the %s folder in the report files", uuid));
+		}
+
 		if (reportUrls.isEmpty()) {
+			log.error("Get share details result: failed, reportId: {}", uuid);
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/service/report/ReportService.java:120]: Failed to ...

File: backend/src/main/resources/application-e2e.yml [hunk 4]
```diff
@@ -15,6 +15,6 @@ jira:
 buildKite:
   url: https://api.buildkite.com
 
-calendarific:
+calendar:
```
Comment (guzhongren) [inline backend/src/main/resources/application-e2e.yml:18]: Delete it

File: backend/src/main/resources/application-local.yml [hunk 5]
```diff
@@ -15,5 +15,5 @@ jira:
 buildKite:
   url: https://api.buildkite.com
 
-calendarific:
+calendar:
```
Comment (guzhongren) [inline backend/src/main/resources/application-local.yml:18]: Delete it

File: backend/src/test/java/heartbeat/repository/FileRepositoryTest.java [hunk 6]
```diff
@@ -60,6 +60,10 @@ class FileRepositoryTest {
 
 	private static final String TEST_UUID = "test-uuid";
 
+	private static final int EXPIRED_DAYS = 7;
```
Comment (guzhongren) [inline backend/src/test/java/heartbeat/repository/FileRepositoryTest.java:63]: add it into test/application.yaml

**Review by guzhongren — APPROVED** [review]

File: backend/src/main/resources/application-local.yml (hunk 5 above)
Comment (zhou-yinyuan) [reply backend/src/main/resources/application-local.yml]: done

File: backend/src/main/resources/application-e2e.yml (hunk 4 above)
Comment (zhou-yinyuan) [reply backend/src/main/resources/application-e2e.yml]: done

File: backend/src/main/java/heartbeat/service/report/ReportService.java (hunk 3 above)
Comment (zhou-yinyuan) [reply backend/src/main/java/heartbeat/service/report/ReportService.java]: done

File: backend/src/main/java/heartbeat/service/report/ReportService.java (hunk 2 above)
Comment (zhou-yinyuan) [reply backend/src/main/java/heartbeat/service/report/ReportService.java]: done

File: backend/src/test/java/heartbeat/repository/FileRepositoryTest.java (hunk 6 above)
Comment (zhou-yinyuan) [reply backend/src/test/java/heartbeat/repository/FileRepositoryTest.java]: I haven't implemented this part. I've tried many methods but none of them worked. You may need to solve it.

File: backend/src/main/java/heartbeat/controller/report/ReportController.java [hunk 7]
```diff
@@ -41,60 +41,60 @@ public class ReportController {
...
-			@PathVariable String uuid,
+			@PathVariable String reportId,
 			@Schema(type = "string", example = "20240310", pattern = "^[0-9]{8}$") @Parameter String startTime,
 			@Schema(type = "string", example = "20240409", pattern = "^[0-9]{8}$") @Parameter String endTime) {
-		log.info("Start to export CSV file_reportType: {}, uuid: {}", reportType.getValue(), uuid);
-		InputStreamResource result = reportService.exportCsv(reportType, uuid, startTime, endTime);
-		log.info("Successfully get CSV file_reportType: {}, uuid: {}, _result: {}", reportType.getValue(), uuid,
+		log.info("Start to export CSV file_reportType: {}, reportId: {}", reportType.getValue(), reportId);
+		InputStreamResource result = reportService.exportCsv(reportType, reportId, startTime, endTime);
+		log.info("Successfully get CSV file_reportType: {}, reportId: {}, _result: {}", reportType.getValue(), reportId,
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/controller/report/ReportController.java:53]: result前不需要下划线，而且result是个比较大的值吧，可以输出result.length之类的

File: backend/src/test/resources/application-test.yml [hunk 8]
```diff
@@ -0,0 +1,2 @@
+heartbeat:
+  expiredDays: ${EXPIRED_DAYS:7}
```
Comment (guzhongren) [inline backend/src/test/resources/application-test.yml:2]: 测试值不能和生产代码的值一样哈

**Review by guzhongren — APPROVED** [review]
Change with the comments

File: backend/src/main/java/heartbeat/controller/report/ReportController.java (hunk 7 above)
Comment (zhou-yinyuan) [reply backend/src/main/java/heartbeat/controller/report/ReportController.java:53]: done

Comment (zhou-yinyuan) [reply backend/src/main/java/heartbeat/controller/report/ReportController.java:53]: result.contentLength方法可以输出长度，但是要抛出IOException，在方法名上throws出去会导致前端报错，用try catch包单独的一个log，感觉没必要，就使用了description

**Pull request merged**